_posts/2022-07-01-towards-using-sforms-for-ontology-engineering-in-openmic.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title:  "Towards Using SForms for Ontology Engineering"
+driveId: 1FLqwJbYEji9sZa-iw2VhEFl4DDSHhzkK
+categories: [Open Mic Session, Open Mic]
+---
+
+On Friday 1th July at 10:30 an open Mic session took place. The topic was \"Towards Using SForms for Ontology Engineering\". Video and presentation included.
+
+{% include googleDrivePlayer.html id=page.driveId %}
+
+The presentation was intended to spark a discussion on how to reuse SForms outside of its original project.
+
+The text presentation is available [here](https://drive.google.com/file/d/1MLrK3070kE-kOTWSh0_2haE9ofyB0kS1/view?usp=sharing).
```

Comment:
link is not public. If you want to make it public, click Share on the file in drive and change from restricted to public

Reply:
Ok I changed it. 
By the way all the other presentations in the folder are also restricted.

Reply:
ok, I'll try to handle it